src/lightbox/LightboxContainer.js

```diff
@@ -62,7 +63,7 @@ class LightboxContainer extends PureComponent<Props, State> {
   handleOptionsPress = () => {
     const options = constructActionSheetButtons();
     const cancelButtonIndex = options.length - 1;
-    const { showActionSheetWithOptions, src, auth } = this.props;
+    const { showActionSheetWithOptions, src, realm, auth } = this.props;
```

Comment:
You can get the realm from the `auth` structure, no need to pass another parameter.

```
export type Auth = {
  realm: string,
  apiKey: string,
  email: string,
};
```

Reply:
got it